kbase-extension/static/kbase/js/common/sdk.js

```diff
@@ -635,14 +635,13 @@ define(['common/props'], (Props) => {
 
```

Comment:
Some minor touching up to the `sdk` module to make my linter happy. There should be more done, like deleting ancient commented out code, but that'll be in another PR.